DentOS_Framework/DentOsTestbedLib/src/dent_os_testbed/lib/traffic/ixnetwork/ixnetwork_ixia_client_impl.py

```diff
@@ -248,6 +248,28 @@ def __update_field(self, field, value):
         field.Auto = False
         field.update(**param)
 
+    def __update_frame_rate(self, config_element, fr_rate_type=None, rate=None, pkt_data=None):
```

Comment:
You don't need fr_rate_type or rate here. You only need pkt_data here

Reply:
what if you want to change in the middle of the test? after the stream has been configured